deploy/01 - Payroll.ts

```diff
@@ -43,7 +43,7 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
       log: true,
     });
 
-    if (deployResult.newlyDeployed && deployResult.transactionHash && network.live) {
+    if (deployResult.newlyDeployed && deployResult.transactionHash && network.live && network.name !== 'rsk') {
```

Comment:
Is this ok? if so ignore this comment

Reply:
newlyDeployed is a flag added from hardahat deploy that is the first time deploying it
transactionHash is  the hash of the transaction where the deploy happend they both exits on RSK

The reason why i added != RSK in this if is because there is no etherscan fork on RSK so the plugin can't verify the contract, we should do that manually